src/internal/kopia/upload.go

```diff
@@ -448,16 +449,21 @@ func streamBaseEntries(
 			return errors.Wrap(err, "getting previous full item path for base entry")
 		}
 
-		// All items have item info in the base backup. However, we need to make
-		// sure we have enough metadata to find those entries. To do that we add the
-		// item to progress and having progress aggregate everything for later.
-		d := &itemDetails{
-			info:         nil,
-			repoPath:     itemPath,
-			prevPath:     prevItemPath,
-			locationPath: locationPath,
+		// Meta files aren't in backup details since it's the set of items the user
+		// sees.
+		if !onedrive.IsMetaFile(itemPath.String()) {
```

Comment:
Just checking the extension feels a bit flaky to me. We should at least check if it a OneDrive file going through this check. Exchange will probably not have `.meta` as a file extension, but we should probably do that check.

Ideal would be to have something like `entity.IsAuxiliary()`, an extra item added to the interface, similar to `IsMeta` on `details.OneDriveInfo`, which we can use and *extend* in the future.